app/src/main/java/com/azuresamples/msalnativeauthandroidkotlinsampleapp/EmailSignInSignUpFragment.kt

```diff
@@ -138,6 +145,7 @@ class EmailSignInSignUpFragment : Fragment() {
                 }
                 is SignUpResult.AttributesRequired,
                 is SignUpResult.PasswordRequired -> {
+                    // AttributesRequired and PasswordRequired only happens after CodeRequired. Please refer to the SignUpCodeFragment for more information.
```

Comment:
This block is for Password required and Attributes required result. I don't think this comment applies